### ISSUE TITLE
Increase the maximum supported kube-apiserver version

### ISSUE THREAD

**Issue by aanm** [cilium/cilium-sysdump#7]
```
INFO -- check the kube-apiserver version --
ERROR the kube-apiserver version is 1.10. We need a version >= 1.7 or <= 1.8.
```

**Comment by aanm** [comment]
This is not addressed for k8s 1.11

**Comment by joestringer** [comment]
Evidently this is not an ongoing problem, or we would have done something about it in the ~3 years since it was filed.
